--- a/jquery.timeago.js	2014-03-13 12:39:52.449750076 +0100
+++ b/jquery.timeago.js	2014-03-13 12:41:56.958243281 +0100
@@ -43,34 +43,37 @@
       allowFuture: false,
       localeTitle: false,
       cutoff: 0,
+      lang: "en",
       strings: {
-        prefixAgo: null,
-        prefixFromNow: null,
-        suffixAgo: "ago",
-        suffixFromNow: "from now",
-        inPast: 'any moment now',
-        seconds: "less than a minute",
-        minute: "about a minute",
-        minutes: "%d minutes",
-        hour: "about an hour",
-        hours: "about %d hours",
-        day: "a day",
-        days: "%d days",
-        month: "about a month",
-        months: "%d months",
-        year: "about a year",
-        years: "%d years",
-        wordSeparator: " ",
-        numbers: []
+        "en": {
+          prefixAgo: null,
+          prefixFromNow: null,
+          suffixAgo: "ago",
+          suffixFromNow: "from now",
+          inPast: 'any moment now',
+          seconds: "less than a minute",
+          minute: "about a minute",
+          minutes: "%d minutes",
+          hour: "about an hour",
+          hours: "about %d hours",
+          day: "a day",
+          days: "%d days",
+          month: "about a month",
+          months: "%d months",
+          year: "about a year",
+          years: "%d years",
+          wordSeparator: " ",
+          numbers: []
+        }
       }
     },

-    inWords: function(distanceMillis) {
+    inWords: function(distanceMillis, lang) {
       if(!this.settings.allowPast && ! this.settings.allowFuture) {
           throw 'timeago allowPast and allowFuture settings can not both be set to false.';
       }

-      var $l = this.settings.strings;
+      var $l = this.settings.strings[lang] || this.settings.strings[this.settings.lang] || this.settings.strings["en"];
       var prefix = $l.prefixAgo;
       var suffix = $l.suffixAgo;
       if (this.settings.allowFuture) {
@@ -180,7 +183,7 @@

     if (!isNaN(data.datetime)) {
       if ( $s.cutoff == 0 || distance(data.datetime) < $s.cutoff) {
-        $(this).text(inWords(data.datetime));
+        $(this).text(inWords(data.datetime, ($(this).attr('lang')) ? $(this).attr('lang') : $t.settings.lang));
       }
     }
     return this;
@@ -200,8 +203,8 @@
     return element.data("timeago");
   }

-  function inWords(date) {
-    return $t.inWords(distance(date));
+  function inWords(date, lang) {
+    return $t.inWords(distance(date), lang);
   }

   function distance(date) {
