--- a/jquery.timeago.js
+++ b/jquery.timeago.js
@@ -31,7 +31,8 @@
       refreshMillis: 60000,
       allowFuture: false,
       localeTitle: false,
-      strings: {
+      lang: "en",
+      strings: { "en": {
         prefixAgo: null,
         prefixFromNow: null,
         suffixAgo: "ago",
@@ -49,10 +50,10 @@
         years: "%d years",
         wordSeparator: " ",
         numbers: []
-      }
+      }}
     },
     inWords: function(distanceMillis) {
-      var $l = this.settings.strings;
+      var $l = this.settings.strings[this.settings.lang] || this.settings.strings["en"] || this.settings.strings;
       var prefix = $l.prefixAgo;
       var suffix = $l.suffixAgo;
       if (this.settings.allowFuture) {
@@ -127,6 +128,7 @@
   }

   function prepareData(element) {
+    $t.settings.lang = ($(element).attr('lang')) ? $(element).attr('lang') : $t.settings.lang;
     element = $(element);
     if (!element.data("timeago")) {
       element.data("timeago", { datetime: $t.datetime(element) });
